bee-storage/bee-storage/src/access/mod.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+//! Access module which form the access layer of the backend which holds the contract of unified database access
```

Comment:
I'ld make it a bit more resistent against renaming:
"This module forms the access layer ....."
Also maybe "contract**s**" (plural?)
I'm not sure: Should bee be BEE or Bee?